modules/boulder/src/main/ts/ephox/boulder/api/Objects.ts

```diff
@@ -33,11 +25,10 @@ const mergeValues = <T>(values: T[], base: T) => {
   );
 };
 
-const mergeErrors = (errors: string[][]): Result<unknown, string[]> => {
-  return Result.error(Arr.flatten(errors));
-};
+const mergeErrors = (errors: string[][]): Result<unknown, string[]> =>
+  Result.error(Arr.flatten(errors));
 
-const consolidate = <T>(objs: Array<Result<T, string[]>>, base: T): Result <T, string> => {
+const consolidate = <T>(objs: Array<Result<T, string[]>>, base: T): Result<T, string> => {
```

Comment:
```suggestion
const consolidate = <T>(objs: Array<Result<T, string[]>>, base: T): Result<T, string[]> => {
```
This is an old issue, but the types here are wrong. It should be this as per the `mergeErrors` return type. IIRC this also requires updating a type in alloy.

Reply:
Done, here and in alloy